ion/integration_test.go

```diff
@@ -366,9 +373,9 @@ func handleEmbeddedDoc(t *testing.T, r Reader) [][]ionItem {
 	return values
 }
 
-func isEmbeddedDoc(an []string) bool {
+func isEmbeddedDoc(an []SymbolToken) bool {
 	for _, a := range an {
-		if a == "embedded_documents" {
+		if a.Text != nil && *a.Text == "embedded_documents" {
 			return true
```

Comment:
IIRC, doesn't this need to be the first annotation? (Rather than anywhere in the annotations list?)

Reply:
The ion test data [files](https://github.com/amzn/ion-tests/blob/05ecd5de78ac6244826cd4e77bae932925c966c1/iontestdata/good/equivs/annotatedIvms.ion#L4) have 'embedded_documents' annotation by itself, it makes sense that it should be the first annotation, updated